packages/regl-worldview/src/commands/GLTFScene.js

```diff
@@ -287,3 +291,19 @@ export default class GLTFScene extends React.Component<Props, {| loadedModel: ?O
     );
   }
 }
+
+// Add unique key to GLTFScene so that it will unmount when the model changes
+export default function GLTFWrapper(props: Props) {
```

Comment:
Did you consider doing something like `reglCommand = getMemoizedCommand(model, regl)` with simple memoization? I think it would be a simpler solution than the wrapper.

If keeping the wrapper function, please set its `displayName` so that React dev tools work better.